src/components/SearchComponents/Search.page.test.tsx

```diff
@@ -297,4 +368,61 @@ describe("Search.page.tsx", () => {
 
     expect(setSearchStartTime).toHaveBeenCalledTimes(1);
   });
+  test("searchResultClicked is called when a search hit is clicked", async () => {
+    const { getByText } = render(<Search {...props} {...resultsProps} />);
+    const description = getByText("lesson title");
+    const user = userEvent.setup();
+    await act(async () => {
+      await user.click(description);
+    });
+
+    expect(searchResultClicked).toHaveBeenCalledTimes(1);
+    expect(searchResultClicked).toHaveBeenCalledWith({
+      analyticsUseCase: null,
+      keyStageSlug: "ks1",
+      keyStageTitle: "Key-stage 1",
+      lessonName: "lesson title",
+      lessonSlug: "lesson-slug",
+      searchFilterOptionSelected: [],
+      searchRank: 1,
+      searchResultCount: 1,
+      searchResultType: "lesson",
+      subjectSlug: "subject-slug",
+      subjectTitle: "subject title",
+      unitName: "lesson title",
+      unitSlug: "topic-slug",
+    });
+  });
+  test("searchResultClicked is called when a pathway hit is clicked", async () => {
+    const { getByText } = render(
+      <Search {...props} {...resultsPropsPathWays} />,
+    );
+    const dropdown = getByText("Select exam board");
+    const user = userEvent.setup();
+    await act(async () => {
```

Comment:
same comment as above: 
https://flyingsquirrel.medium.com/when-should-i-use-act-in-react-testing-library-d7dd22a3340e

Reply:
Also might be worth double checking that this is still passing with a .only, for some reason it failed when I did